docs/topics/Gateway.md

```diff
@@ -833,6 +846,45 @@ The inner `d` key is a boolean that indicates whether the session may be resumab
 
 `APPLICATION_COMMAND_PERMISSIONS_UPDATE` event, sent when an application command's permissions are updated. The inner payload is an [application command permissions](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-permissions-object-guild-application-command-permissions-structure) object.
 
+### Auto Moderation
+
+All auto moderation related events are currently only sent to bot users which have the `MANAGE_GUILD` permission.
+
+#### Auto Moderation Rule Create
+
+Sent when a rule is created. The inner payload is an [auto moderation rule](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-rule-object) object.
+
+#### Auto Moderation Rule Update
+
+Sent when a rule is updated. The inner payload is an [auto moderation rule](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-rule-object) object.
+
+#### Auto Moderation Rule Delete
+
+Sent when a rule is deleted. The inner payload is an [auto moderation rule](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-rule-object) object.
+
+#### Auto Moderation Action Execution
+
+Sent when an rule is triggered and an action is executed (e.g. message is blocked).
+
+###### Auto Moderation Action Execution Event Fields
```

Comment:
Will a `user_id` be added to these fields? Currently, there's no way to know which user tried to send a message if the only action was blocking the message, even if the action was timeout- the way to get which user it was is kinda a workaround.

Reply:
great point, we'll add the user_id 👍